### ISSUE TITLE
go get -u github.com/gobuffalo/packr/v2/...

### ISSUE THREAD

**Issue by dalu** [gobuffalo/packr#133]
```
go get -u github.com/gobuffalo/packr/v2/...
package github.com/gobuffalo/packr/v2/...: github.com/gobuffalo/packr/v2/...: invalid import path: malformed import path "github.com/gobuffalo/packr/v2/...": double dot
```

```
go version go1.11.3 linux/amd64
```

**Comment by jcburley** [comment]
Go 1.11.3 has a bug that might explain this.

Try upgrading to 1.11.4 (which was released shortly after 1.11.3) or falling back to 1.11.2.

**Comment by dalu** [comment]
Hmm it's not packaged for Archlinux yet.
I'll close this then. Thanks.
